127.Word Ladder

```diff
@@ -0,0 +1,43 @@
+//Time: O(n^2), Space: O (n)
+class Solution {
+    public:
+        int ladderLength(string beginWord, string endWord, vector<string>& wordList) 
+        {
+            //pair<string, int> stands for a string and its current level in the ladder
```

Comment:
So you are using comment here to tell people what does pair<string, int> mean. However, if you use
`using possibleStep = pair<string, int>` will be better.